ci/views.py

```diff
@@ -493,6 +492,216 @@ def identities(request):
     context['form'] = form
     return render(request, 'ci/identities.html', context)
 
+def user_management_detail(request, identity):
+    idApi = IdentityStoreApiClient(
+        api_url=request.session["user_tokens"]["SEED_IDENTITY_SERVICE"]["url"],  # noqa
+        auth_token=request.session["user_tokens"]["SEED_IDENTITY_SERVICE"]["token"]  # noqa
+    )
+    hubApi = HubApiClient(
+        api_url=request.session["user_tokens"]["HUB"]["url"],  # noqa
+        auth_token=request.session["user_tokens"]["HUB"]["token"]  # noqa
+    )
+    sbmApi = StageBasedMessagingApiClient(
+        api_url=request.session["user_tokens"]["SEED_STAGE_BASED_MESSAGING"]["url"],  # noqa
+        auth_token=request.session["user_tokens"]["SEED_STAGE_BASED_MESSAGING"]["token"]  # noqa
+    )
+    msApi = MessageSenderApiClient(
+        api_url=request.session["user_tokens"]["SEED_MESSAGE_SENDER"]["url"],  # noqa
+        auth_token=request.session["user_tokens"]["SEED_MESSAGE_SENDER"]["token"]  # noqa
+    )
+    messagesets_results = sbmApi.get_messagesets()
+    messagesets = {}
+    schedules = {}
+    choices = []
+    linked_to = []
+    operator_id = []
+   
+    
+   
+    for messageset in messagesets_results["results"]:
+        messagesets[messageset["id"]] = messageset["short_name"]
+        schedules[messageset["id"]] = messageset["default_schedule"]
+        choices.append((messageset["id"], messageset["short_name"]))
+
+    results = idApi.get_identity(identity)
+    sbm_filter = {
+        "identity": identity
+    }
+    subscriptions = sbmApi.get_subscriptions(params=sbm_filter)
+
+    if request.method == "POST":
```

Comment:
We don't need this here, the template doesn't have any forms posting to this view.